src/control/control.h

```diff
@@ -125,9 +125,12 @@ class ControlDoublePrivate : public QObject {
     void valueChangeRequest(double value);
 
   private:
+    // The defaultValue is adopted as initial value as well
```

Comment:
// If bPersist is true and the configuration object is not null, 
// the initial value for this control is the value stored in the configuration.
// Otherwise, defaultValue is used.